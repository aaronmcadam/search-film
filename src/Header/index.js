import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`padding-left: 1rem;`;
const AppName = styled.h1`font-size: 3rem;`;
const Intro = styled.p`font-size: 1.5rem;`;

export default function Header() {
  return (
    <Wrapper>
      <AppName>SearchFilm</AppName>
      <Intro>Start typing to get started!</Intro>
    </Wrapper>
  );
}
