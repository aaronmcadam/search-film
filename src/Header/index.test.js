import React from 'react';
import { shallow } from 'enzyme';
import Header from '.';

const setup = () => {
  const wrapper = shallow(<Header />);

  return {
    wrapper,
  };
};

test('renders', () => {
  const { wrapper } = setup();

  expect(wrapper).toMatchSnapshot();
});
