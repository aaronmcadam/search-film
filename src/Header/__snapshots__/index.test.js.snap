// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <styled.div>
    <styled.h1>
        SearchFilm
    </styled.h1>
    <styled.p>
        Start typing to get started!
    </styled.p>
</styled.div>,
  "nodes": Array [
    <styled.div>
      <styled.h1>
            SearchFilm
      </styled.h1>
      <styled.p>
            Start typing to get started!
      </styled.p>
</styled.div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Header />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <styled.div>
          <styled.h1>
                    SearchFilm
          </styled.h1>
          <styled.p>
                    Start typing to get started!
          </styled.p>
</styled.div>,
        "_debugID": 2,
        "_renderedOutput": <styled.div>
          <styled.h1>
                    SearchFilm
          </styled.h1>
          <styled.p>
                    Start typing to get started!
          </styled.p>
</styled.div>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Header />,
}
`;
